backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -0,0 +1,35 @@
+package app.ehrenamtskarte.backend.application.webservice
+
+import app.ehrenamtskarte.backend.application.database.repos.EakApplicationRepository
+import app.ehrenamtskarte.backend.application.webservice.schema.create.BlueCardApplication
+import app.ehrenamtskarte.backend.application.webservice.schema.create.GoldenEakCardApplication
+import com.expediagroup.graphql.annotations.GraphQLDescription
+import graphql.schema.DataFetchingEnvironment
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class EakApplicationMutationService {
+
+    @GraphQLDescription("Stores a new blue digital EAK")
+    fun addBlueEakApplication(
+        regionId: Int,
+        application: BlueCardApplication,
+        dataFetchingEnvironment: DataFetchingEnvironment
+    ): Boolean {
+        EakApplicationRepository.addBlueEakApplication(regionId, application, dataFetchingEnvironment)
+        return true
```

Comment:
Just personal interest: is there a reason we return `true` here? Does GraphQL require something to be returned?

Reply:
I think GraphQl doesn't require any return value (but I don't know). ^^

Reply:
I did some copy pasta from the CardMutationService.